set.mm

```diff
@@ -422891,24 +423014,30 @@ varying definitions (some start from 0, others start from 1), but
        details).  Similarly, ` F/_ x A ` is read ` x ` is not free in (class)
        ` A ` , see ~ df-nfc .</li>
 
-       <li>"$d x y $." should be read "Assume x and y are distinct
+       <li>"$d ` x y ` $." should be read "Assume ` x ` and ` y ` are distinct
        variables."</li>
 
-       <li>"$d x ` ph ` $." should be read "Assume x does not occur in phi $."
-       Sometimes a theorem is proved using
-       ` F/ x ph `  ( ~ df-nf ) in place of
-       "$d ` x ph ` $." when a more general result is desired;
+       <li>"$d ` ph x ` $." should be read "Assume ` x ` does not occur in
+       ` phi `."  Sometimes a theorem is proved using ` F/ x ph `  ( ~ df-nf )
+       in place of "$d ` ph x ` $." when a more general result is desired;
        ~ ax-5 can be used to derive the $d version.  For an example of
        how to get from the $d version back to the $e version, see the
        proof of ~ euf from ~ eu6 .</li>
 
-       <li>"$d x A $." should be read "Assume x is not a variable occurring in
-       class A."</li>
+       <li>"$d ` A x ` $." should be read "Assume ` x ` is not a variable
+       occurring in class ` A `."</li>
+
+       <li>"$d ` A x ` $.  $d ` ps x ` $.
+       $e |- ` ( x = A -> ( ph <-> ps ) ) ` $." is an idiom often used instead
+       of explicit substitution, meaning "Assume ` psi ` results from the
+       proper substitution of ` A ` for ` x ` in ` phi `."</li>
```

Comment:
Can you take that opportunity to add something like: "Therefore, we often use the term "implicit substitution" for such a hypothesis." ? Indeed, that term is used in several parts of set.mm ?

Reply:
Yes, good idea.

Reply:
Done.